src/utils/daos_hdlr.h

```diff
@@ -89,6 +89,7 @@ struct cmd_args_s {
 	char			*dst;		/* --dst path for fs copy */
 	daos_cont_layout_t	type;		/* --type cont type */
 	daos_oclass_id_t	oclass;		/* --oclass object class */
+	uint32_t		mode;		/* --posix consistency mode */
```

Comment:
This could be from the bad merge I mentioned above?  I didn't think this PR had landed yet.